### DISCUSSION TITLE
Stream Deck: button frame color mobile app

### DISCUSSION THREAD

**Discussion by Christian-548376** [elgatosf/elgato#36] [Help]
is there a way to change the button frame color in the **mobile app**. I need it to be darker or even transparent/translucent.

![iPhone](https://github.com/elgatosf/elgato/assets/136122436/71e83e7d-3787-4a76-9e5b-1ab8a748e816)

I have Stream Deck mobile 2.0 (Pro) and Mac version 6.3

**Comment by GeekyEggo** [comment]
Hey @Christian-548376, for assistance with the Stream Deck applications, including Stream Deck Mobile, please refer to our wonderful [community Discord](https://discord.gg/YW6aSEvS) or official [Elgato Support](http://e.lga.to/help).
